presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -765,7 +775,7 @@ private void testUnsupportedDataType(String databaseDataType)
                 TIMESTAMP_WITH_TIME_ZONE,
                 DateTimeFormatter.ofPattern("'TIMESTAMP '''yyyy-MM-dd HH:mm:ss.SSS VV''")::format,
                 // PostgreSQL does not store zone, only the point in time
-                zonedDateTime -> zonedDateTime.withZoneSameInstant(ZoneId.of("UTC")));
+                zonedDateTime -> zonedDateTime == null ? null : zonedDateTime.withZoneSameInstant(ZoneId.of("UTC")));
```

Comment:
IMO it's safe to modify `io.prestosql.tests.datatype.DataType#toPrestoQueryResult` and handle nulls there, just as we do in `io.prestosql.tests.datatype.DataType#toLiteral`:
```
public Object toPrestoQueryResult(T inputValue)
{
    if (inputValue == null) {
        return null;
    }
    return toPrestoQueryResult.apply(inputValue);
}
```

then the change here is not needed

Reply:
done